helm/identity-account-mapper/account-mapper/templates/deployment.yaml

```diff
@@ -22,9 +22,12 @@ spec:
     spec:
       initContainers:
         #During this Pod's initialization, check that operationsmysql service is up and running before starting this pod
-        - name: check-operationsmysql-ready
-          image: busybox:latest
-          command: [ 'sh', '-c','until nc -vz operationsmysql 3306; do echo "Waiting for operationsmysql service"; sleep 2; done;' ]
+        - name: check-db-service
+          image: {{.Values.initContainers.image}}
+          command: [ 'sh', '-c','until nc -vz {{.Values.datasource.host}} {{.Values.datasource.port}}; do echo "Waiting for database service"; sleep 2; done;' ]
```

Comment:
why cant you make the entire init container section configurable from values.yml

Reply:
I think the below lines are doing it. then the above lines may not be needed

Reply:
Yes, the lines below are doing it, but the problem is that if these init containers are set from ph-ee-engine and need to add one more initcontainer from another chart like g2p-sandbox, only changes from g2p-sandbox are applied it will remove ph-ee-engine initcontainers.